docs/topics/ref_known-issues-7-2-0.adoc

```diff
@@ -0,0 +1,43 @@
+
+.{ProductShortName} [CLI] analysis using the Containerless CLI fails on Windows
+
+{ProductShortName} [CLI] analysis using the Containerless CLI fails on Microsoft Windows and returns a `provider does not have dependency capability` error. This failure is caused by the existence of the `.metadata` folder located inside the folder where the analysis is run. Therefore, the workaround is to delete this folder. This issue is planned to be resolved in {ProductShortName} 7.2.1. link:https://issues.redhat.com/browse/MTA-4416[(MTA-4416)]
+
+.{ProductShortName} CLI analysis is extremely slow on Windows
+
+{ProductShortName} CLI analysis does not function as expected on Microsoft Windows, with analysis using container runtimes being extremely slow on Windows. This issue is planned to be resolved in {ProductShortName} 7.2.1. link:https://issues.redhat.com/browse/MTA-3667[(MTA-3667)]
+
+.Analysis of the daytrader application does not show dependencies
+
+The analysis of the daytrader application does not show dependencies. This failure to display dependencies is caused by the {ProductShortName} Operator using Maven 3.6, which sometimes fails to fetch dependencies. This issue is planned to be resolved in {ProductShortName} 7.2.1. link:https://issues.redhat.com/browse/MTA-4169[(MTA-4169)]
+
+.{ProductShortName} CLI bulk analysis does not function on Windows
+
+{ProductShortName} CLI bulk analysis, performed by using the `--bulk` option, does not function on Microsoft Windows. This issue is planned to be resolved in {ProductShortName} 7.2.1. link:https://issues.redhat.com/browse/MTA-4307[(MTA-4307)]
+
+.{ProductShortName} Containerless CLI does not fire XML custom rules
+
+The {ProductShortName} Containerless CLI does not fire XML custom rules, and the `analyze-bin` command is not raising violations coming from custom rules in XML format. link:https://issues.redhat.com/browse/MTA-4097[(MTA-4097)
+
+.Binary analysis fails with incorrect Maven credentials
+
+{ProductShortName} CLI analysis fails with a generic error when incorrect Maven credentials are used. The failure occurs because the analyzer is unable to report detailed error messages. link:https://issues.redhat.com/browse/MTA-3714[(MTA-3714)]
+
+.{ProductShortName} CLI returns confusing error messages when Python is not installed on Windows
+
+{ProductShortName} CLI returns two confusing error messages of `failed to validate flags` and `exit status 9009` when Python is not installed on Microsoft Windows. This issue is planned to be resolved in {ProductShortName} 7.2.1. link:https://issues.redhat.com/browse/MTA-4319[(MTA-4319)]
```

Comment:
I apologize; I'm not sure why this bug was still open with a release notes label. I just closed it as it's obsolete. You can remove this section entirely